CONTRIBUTING.md

```diff
@@ -93,12 +93,12 @@ Here are a few examples:
 
 * Create one file `changelog/{type}_{some_description}.md`, where `type` is `new` (New feature), `fix` or `change`, and `some_description` is unique to avoid conflicts. Task `changelog:fix` (or `:new` or `:change`) can help you.
 * Mark it up in [Markdown syntax][6].
-* The entry line should start with `* ` (an asterisk and a space).
+* The entry line should one line starting with `* ` (an asterisk and a space).
```

Comment:
How about "The entry should be a single line, starting with ...".

Reply:
... should **be on** one line starting ...

Reply:
Sure, I updated it. Thank you!